clisops/ops/subset.py

```diff
@@ -110,8 +112,17 @@ def subset(
 
     """
     # Convert all inputs to Xarray Datasets
+    if ds is None:
+        raise MissingParameterValue
```

Comment:
This should be raised by ` args = utils.map_params(ds, time, area, level)` on new line 128 if `ds is None`, so I'm wondering if it needs to be implemented twice?

Reply:
You're right, it doesn't need to be there. I'll see what happens if I take it out.